packages/hash/api/jest.config.cjs

```diff
@@ -5,6 +5,7 @@ module.exports = {
     "**/*.{c,m,}{j,t}s{x,}",
     "!**/node_modules/**",
     "!**/dist/**",
+    "!**/src/collab/**",
```

Comment:
I've excluded collab from code coverage, to fix this CI error I was encountering:
<img width="1124" alt="image" src="https://user-images.githubusercontent.com/42802102/198607529-dddd5c2a-64bc-41a5-b95c-537beca92e92.png">